src/components/CommonGridBasedForm/index.js

```diff
@@ -51,6 +52,9 @@ const CommonGridBasedForm = ({ fields, buttons, responseError, heading, onSaveSu
           />
         );
 
+      case LOADER:
```

Comment:
Why Loader is there in getFields ??

Reply:
Was there if someone wants to switch to loader with a field. Removing, as it was for test purposes.